libs/server/repository/infrastructure/src/lib/bitbucket/mappers/map-bitbucket-pr-details.ts

```diff
@@ -0,0 +1,19 @@
+import { plainToClass } from '@marcj/marshal';
+import { PrEntity } from '@pimp-my-pr/server/repository/core/domain';
+import { BitbucketPrDetailsEntity } from '../domain/entities/bitbucket-pr-details.entity';
+import { mapBitbucketAccount } from './map-bitbucket-account';
+
+export const mapBitbucketPrDetails = (pr: BitbucketPrDetailsEntity): PrEntity =>
+  plainToClass(PrEntity, {
+    ...pr,
+    additions: pr.diff.map(diff => diff.lines_added).reduce((prev, curr) => prev + curr),
+    changedFiles: pr.diff.length,
+    createdAt: pr.created_on,
+    deletions: pr.diff.map(diff => diff.lines_removed).reduce((prev, curr) => prev + curr),
```

Comment:
maybe calculate deletions and additions on the beginning inside one iteration through the diff array.